rpcserver.go

```diff
@@ -3964,6 +3964,13 @@ func (r *rpcServer) ListChannels(ctx context.Context,
 			continue
 		}
 
+		info, err := r.GetNodeInfo(ctx, &lnrpc.NodeInfoRequest{})
```

Comment:
This isn't quite right. I'd suggest updating the test `testListChannels` to test the change. From there you should be able to see what's missing.

Reply:
@yyforyongyu could you tell me how I can run `testListChannels`? I only seem to be able to run tests enabled in `lnd_test_list_on_test.go` via `make itest icase=....` but `testListChannels` is in `list_on_test.go` which fails when run like this.

Reply:
You can run `make itest temptest=true icase=...`.

Reply:
Thank you, couldn't figure out this detail.